### PR TITLE
add the "generateImports" function (used in "generateTSCaller")

### PR DESCRIPTION
**Pull request by Elli610**
related to #207

### CODE REVIEW

**Review by 0xSwapFeeder — APPROVED** [review]

File: packages/massa-proto-cli/src/ts-gen.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { ProtoFile } from './protobuf';
+
+function generateImports(protoFile: ProtoFile, helperRelativePath): string {
+  let content = `import { ${protoFile.funcName}Helper } from "${helperRelativePath}";\n\n`;
+  content += `import { Account } from 'wallet-provider';`;
```
Comment (gregLibert) [inline packages/massa-proto-cli/src/ts-gen.ts:5]: Why do you need the account ?

File: packages/massa-proto-cli/src/ts-gen.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * @returns The typescript code to import the helper and the Account interface
+ */
+function generateImports(protoFile: ProtoFile, helperRelativePath): string {
+  let content = `import { ${protoFile.funcName}Helper } from "${helperRelativePath}";
+export interface ITransactionDetails {
+  operationId: string;
+}
+
+`;
+  return content;
```
Comment (gregLibert) [inline packages/massa-proto-cli/src/ts-gen.ts:18]:
```suggestion
 return `import { ${protoFile.funcName}Helper } from "${helperRelativePath}";
export interface TransactionDetails {
  operationId: string;
}\n`;
```

File: packages/massa-proto-cli/src/ts-gen.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * Generates the typescript code to import the helper file
+ *
+ * @param protoFile - The proto file data
+ * @param helperRelativePath - The relative path to the previously generated helper
+ *
+ * @returns The typescript code to import the helper and the Account interface
+ */
+function generateImports(protoFile: ProtoFile, helperRelativePath): string {
```
Comment (gregLibert) [inline packages/massa-proto-cli/src/ts-gen.ts:11]: Your not only importing thing, but also exporting the interface. You should remove that part of the code in another function or update the function name if you want to do both. The second approach is messier becasue functionaly you are doing two different things

**Review by gregLibert — CHANGES_REQUESTED** [review]

**Comment by gregLibert** [comment]
@Elli610 in my commit I:
- clarify the export function name (interface is a little too vague)
- set the type of `helperRelativePath`, but maybe it's not a string
- add a description for `generateTransactionDetailsExport`
- remove new lines. Using them where they will be used will make code and tests cleaner
- remove useless variable content
- improve the documentation of the functino by explaining what we are doing from a functional point of view instead of a technical one

Can you take all that into account for the other reviews ?

**Pull request closed without merging**